haunting-of-the-hill-house.html

```diff
@@ -38,10 +38,10 @@
             <a href="#footer" id="navigation"><img src="images/reviews_logo_wht.svg" alt="reviews&more logo"
                     class="logo"></a>
             <div class="menu-right">
-                <div class="search-container"><input id="search" placeholder="Keresés" name="search" type="search"
-                        autocomplete="off" />
+                <div class="search-container"><label for="search"></label><input id="search" placeholder="Keresés"
```

Comment:
Az input tagben lehetne akadalymentesitö eszközöket hasznalni. Pl:
- aria-label="Keresés az oldalon" 
- role="searchbox"

Illetve adott esetben az aria-labelledby is hasznalhato, ha a label-t az az input elött emiatt hoztad letre, viszont az még nem teljes (lasd elözö komment)

Reply:
Közben feltünt, hogy az input tagen belül hasznalod a type="search" attributomot. Utana neztem, hogy ez hogy kombinalhato ez en javaslataimmal. A következö oldal szerint a te megoldasod es az en ket javaslatom egyenertekü:
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/searchbox_role

Ettöl függetlenül erdemes lenne kiprobalni felolvaso programmal, hogy mi a különbseg. 

Ez nekem is hasznos info... :)